code/game/objects/items/plushes.dm

```diff
@@ -33,7 +33,8 @@
 
 /obj/item/toy/plush/Initialize()
 	. = ..()
-	AddComponent(/datum/component/squeak, squeak_override)
+	if(squeak_override)
+		AddComponent(/datum/component/squeak, squeak_override)
```

Comment:
You're changing functionality here, it's fine for squeak_override to be null

Reply:
some plushies dont have squeak_overrides

Reply:
And that's fine, if it's null it uses the default value which is intentional